exporter/datadogexporter/metrics_translator.go

```diff
@@ -40,9 +42,30 @@ func getTags(labels pdata.StringMap) []string {
 	return tags
 }
 
+// isCumulativeMonotonic checks if a metric is a cumulative monotonic metric
+func isCumulativeMonotonic(md pdata.Metric) bool {
+	switch md.DataType() {
+	case pdata.MetricDataTypeIntSum:
+		return md.IntSum().AggregationTemporality() == pdata.AggregationTemporalityCumulative &&
+			md.IntSum().IsMonotonic()
+	case pdata.MetricDataTypeDoubleSum:
+		return md.DoubleSum().AggregationTemporality() == pdata.AggregationTemporalityCumulative &&
+			md.DoubleSum().IsMonotonic()
+	}
```

Comment:
@mx-psi - Am I reading this correctly that if the client is already exporting metrics to the collector as `AGGREGATION_TEMPORALITY_DELTA` then they will be sent to datadog as is?

Reply:
Yes, the cases for the current types are:
- For delta metrics the raw value is sent as a gauge
- For cumulative non-monotonic metrics, the raw value is sent as a gauge
- For cumulative monotonic metrics, the rate between the previous datapoint and the next one is calculated and is sent as a rate (this is what this PR does)

Reply:
> Would you expect it to be sent differently?

Nope, that makes sense.  I just wanted to confirm because we have configured our apps to export metrics via OTLP using `DeltaExportKindSelector`.  This was to work around #1688 but also because we have some internal metrics pipelines that rely on metrics being emitted as deltas as well.
```go
	pusher := push.New(
		basic.New(
			simple.NewWithExactDistribution(),
			metric.DeltaExportKindSelector(),
		),
```

Thanks for the quick reply!

Reply:
Awesome, glad this makes sense to you :)